package.json

```diff
@@ -46,6 +46,7 @@
     "@lavamoat/lavapack@^3.3.0": "patch:@lavamoat/lavapack@npm%3A3.3.0#./.yarn/patches/@lavamoat-lavapack-npm-3.3.0-5a61b5374d.patch",
     "@types/mocha@^10.0.1": "patch:@types/mocha@npm:10.0.1#.yarn/patches/@types-mocha-npm-10.0.1-7c94e9e170.patch",
     "clet@^1.0.1": "patch:clet@npm%3A1.0.1#./.yarn/patches/clet-npm-1.0.1-8523231bdc.patch",
+    "find-babel-config": "^2.0.0",
```

Comment:
Can we upstream this?

Reply:
I can try!

Reply:
I have opened an upstream PR

Reply:
We should merge this now though, we can remove the resolution if my upstream PR is merged.